tests/test_http.py

```diff
@@ -227,10 +228,26 @@ def test_dav_as_local(self):
         self.assertEqual(ResourcePath(local_path).read(), contents)
         os.remove(local_path)
 
+    def test_dav_size(self):
+        # Size of an inexistent file must raise
```

Comment:
"a nonexistent file"  (also, period at end of comment)

Reply:
Fixed.